src/extensions/default/CSSCodeHints/styles/brackets-css-hints.css

```diff
@@ -28,3 +28,22 @@
 .dark .brackets-css-hints .matched-hint {
     color: #ccc;
 }
+
+.brackets-css-hints.left-margin {
+    margin-left: 8px;
+}
+
+.brackets-css-hints .color-swatch {
+    border-radius: 2px;
+    width: 12px;
+    height: 12px;
+    float: left;
+    margin-top: 2px;
+    margin-left: 4px;
+    margin-right: -8px;
+    box-shadow: inset 0 0 2px rgba(0, 0, 0, 0.24);
+}
```

Comment:
Would be nice if these rules were added to core styles folder so other code hints, and maybe even inline color editor, could use them.